src/ducks/reimbursements/Caption.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import cx from 'classnames'
+
+import styles from 'ducks/reimbursements/Reimbursements.styl'
+
+const Caption = props => {
+  const { className, ...rest } = props
+
+  return <p className={cx(styles.Caption, className)} {...rest} />
+}
+
+export default React.memo(Caption)
```

Comment:
this component does not need React.memo in my opinion. It's not too heavy.

Source: https://dmitripavlutin.com/use-react-memo-wisely/